### DISCUSSION TITLE
Maps not sent to the site/removed from the site should be marked as Local

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#19694] [Q&A] [answered]
Currently maps that are created locally (the most common example being created in stable and imported into lazer) do not have any map status. It would be worth assigning them a new type of status "Local". But if you edit the map in the lazer, then it becomes Local.
![image](https://user-images.githubusercontent.com/59089206/183987265-b36e2e07-f919-4b28-8aad-0475d8dd0e1c.png)

**Comment by peppy** [comment, marked as answer]
this is already the case for all new beatmaps.
